templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -80,6 +80,8 @@ below, and add more to your repertoire as needed.
 
 * **Narrow to all messages**: `Esc` or `Ctrl` + `[` — Shows all unmuted messages.
 
+* **Narrow to current compose box context**: `Ctrl` + `\` — Narrows to the current stream/topic/PM in the compose box.
```

Comment:
"/PM thread"?

Reply:
Maybe we can call this **Narrow to compose target**?

Reply:
Handled.